### PR TITLE
fix(DataSeriesFormItemModal): larger form modal

### LINKED ISSUE #3468: [DataSeriesFormItemModal] UI threshold value modal css issues
### What package is this for?

- [x] React
- [ ] Angular

### Describe the bug

 - Threshold number input: +- symbols and value are squeezed together because there is not enough space in the modal

### To Reproduce

1. Go to any dashboard -> Edit dashboard: eg. https://dashboard-dev.connectedproducts.internetofthings.ibmcloud.com/monitor/Compresser/editSummary/227605/2
2. Add a 'Data Table' card
3. Select a 'Data item'
4. Edit newly added data item
5. In the 'Customize data series' modal, click 'Add threshold'
6. Enter a value in the number input
7. Notice the overlap between the value and +- symbols

### Expected behavior

Input value and symbols should be correctly displayed

### Environment/versions:

- OS: MacOS
- Browser: Chrome
- `carbon-addons-iot-react` version: 2.150.1-next.1

### Additional context

image

### PR DESCRIPTION
**Pull request by cosminclinciu1**
Closes #3468

**Summary**

- Increased modal size from xs -> md
- Modified test case for the md modal size

**Acceptance Test (how to verify the PR)**

 - Go to any dashboard -> Edit dashboard: eg. https://dashboard-dev.connectedproducts.internetofthings.ibmcloud.com/monitor/Compresser/editSummary/227605/2
 - Add a 'Data Table' card
- Select a 'Data item'
- Edit newly added data item
- In the 'Customize data series' modal, click 'Add threshold'
- Enter a value in the number input
- Modal size is larger (md), with no overlap in the number input between the value and +- symbols

**Regression Test (how to make sure this PR doesn't break old functionality)**

- tests here

**Things to look for during review**

- [ ] Make sure all references to `iot` or `bx` class prefix is using the prefix variable
- [ ] (React) All major areas have a `data-testid` attribute. New test ids should have test written to ensure they are not changed or removed.
- [ ] UI should be checked in RTL mode to ensure the proper handling of layout and text.
- [ ] All strings should be translatable.
- [ ] The code should pass a11y scans (The storybook a11y knob should show no violations). New components should have a11y test written.
- [ ] Unit test should be written and should have a coverage of 90% or higher in all areas.
- [ ] All components should be passing visual regression test. For new styles or components either a visual regression test should be written for all permutations or the base image updated.
- [ ] Changes or new components should either write new or update existing documentation.
- [ ] PR should link and close out an existing issue

### CODE REVIEW

**Review by herleraja — CHANGES_REQUESTED** [review]
https://next.carbon-addons-iot-react.com/?path=/story/2-watson-iot-experimental-%E2%98%A2%EF%B8%8F-cardeditor-cardeditformitems-thresholdformitem--example-with-initial-values
image


https://deploy-preview-3469--carbon-addons-iot-react.netlify.app/?path=/story/2-watson-iot-experimental-%E2%98%A2%EF%B8%8F-cardeditor-cardeditformitems-thresholdformitem--example-with-initial-values

image

Looks like its creating some issues.

File: packages/react/src/components/CardEditor/CardEditForm/_card-edit-form.scss [hunk 1]
```diff
@@ -174,7 +174,7 @@
       width: 50%;
     }
     &--item-end {
-      width: 100%;
+      width: 50%;
```
Comment (herleraja) [inline packages/react/src/components/CardEditor/CardEditForm/_card-edit-form.scss:177]: I'm not sure if we need these changes

File: packages/react/src/components/CardEditor/CardEditForm/_card-edit-form.scss [hunk 2]
```diff
@@ -191,6 +191,7 @@
     &--item-dropdown {
       margin-right: $spacing-05;
       max-width: 6rem;
+      min-width: 15%;
```
Comment (herleraja) [inline packages/react/src/components/CardEditor/CardEditForm/_card-edit-form.scss:194]: any specific reason we need this?

**Comment by cosminclinciu1** [comment]
> https://next.carbon-addons-iot-react.com/?path=/story/2-watson-iot-experimental-%E2%98%A2%EF%B8%8F-cardeditor-cardeditformitems-thresholdformitem--example-with-initial-values image
> 
> https://deploy-preview-3469--carbon-addons-iot-react.netlify.app/?path=/story/2-watson-iot-experimental-%E2%98%A2%EF%B8%8F-cardeditor-cardeditformitems-thresholdformitem--example-with-initial-values
> 
> image
> 
> Looks like its creating some issues.

The idea was to make the dropdowns a bit wider and the number input smaller. This is because the modal is larger, but only the number input gets wider with the initial css, while the dropdowns remain very small.

Reverted the changes, as it works fine anyway

**Review by herleraja — APPROVED** [review]
LGTM

**Pull request merged**